client/secureclient/src/main/java/com/linkedin/openhouse/client/ssl/HousetablesApiClientFactory.java

```diff
@@ -50,6 +51,9 @@ public ApiClient createApiClient(@NonNull String baseUrl, String token, String t
       throws MalformedURLException, SSLException {
     WebClient webClient = createWebClient(baseUrl, token, truststoreLocation);
     ApiClient apiClient = new ApiClient(webClient);
+    if (token != null && !token.isEmpty()) {
+      apiClient.addDefaultHeader(HttpHeaders.AUTHORIZATION, String.format("Bearer %s", token));
+    }
```

Comment:
```suggestion
```
We don't have authentication support in housetables service. Should we remove this?

Reply:
Done